<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="bagPath" default="/seerep/seerep-data/9a3490e2-ae1e-4436-81f5-f996e9e14d95_processed.bag"/>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/"/>
    <arg name="projectFrameId" default="map"/>
    <arg name="projectName" default="TestDump"/>
    <arg name="projectUuid" default=""/>
    <arg name="topicImage" default="/ai_test_field/edge/hsos/sensors/zed2i/zed_node/rgb/image_rect_color/compressed, /ai_test_field/edge/dfki/sensors/stereo/left/image_color/compressed"/>
    <arg name="topicPc2" default="/ai_test_field/edge/hsos/sensors/ouster/points, /ai_test_field/edge/hsos/sensors/rslidar_points, /ai_test_field/edge/hsos/sensors/velodyne_points /ai_test_field/edge/hsos/sensors/zed2i/zed_node/point_cloud/cloud_registered"/>
    <arg name="topicCameraIntrinsics" default="/ai_test_field/edge/hsos/sensors/zed2i/zed_node/rgb/camera_info, /ai_test_field/edge/dfki/sensors/stereo/left/camera_info"/>
    <arg name="topicTf" default="/tf"/>
    <arg name="topicTfStatic" default="/tf_static"/>
    <arg name="topicDetection" default="/detection"/>
    <arg name="maxViewingDistance" default="50.0, 50.0"/>

    <node name="seerep_ros_communication_rosbag_dumper" pkg="seerep_ros_comm" type="rosbag_dumper_aitf" output="screen">
        <rosparam param="bagPath" subst_value="True">$(arg bagPath)</rosparam>
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="projectFrameId" subst_value="True">$(arg projectFrameId)</rosparam>
        <rosparam param="projectName" subst_value="True">$(arg projectName)</rosparam>
        <rosparam param="projectUuid" subst_value="True">$(arg projectUuid)</rosparam>
        <rosparam param="topicImage" subst_value="True">[$(arg topicImage)]</rosparam>
        <rosparam param="topicPc2" subst_value="True">[$(arg topicPc2)]</rosparam>
        <rosparam param="topicCameraIntrinsics" subst_value="True">[$(arg topicCameraIntrinsics)]</rosparam>
        <rosparam param="topicTf" subst_value="True">$(arg topicTf)</rosparam>
        <rosparam param="topicTfStatic" subst_value="True">$(arg topicTfStatic)</rosparam>
        <rosparam param="maxViewingDistance" subst_value="True">[$(arg maxViewingDistance)]</rosparam>
    </node>
</launch>
